wbooks/src/app/screens/book-detail/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import {  
+  View, 
+  Image,
+  Text,
+  TouchableOpacity
+} from 'react-native';
+import PropTypes from 'prop-types';
+
+import styles from './styles';
+
+class BookDetail extends Component {
+	 
+	render() {
+
+    const book = this.props.navigation.getParam('book');
+
+		return (
+      <View style={styles.container}>
+        <View style={styles.cardDetail}>
+          <View style={styles.cardContainer}>
+            <Image style={styles.image}
+            source={{uri: book.image_url}} />
+            <View style={styles.textContainer}>
+              <Text style={styles.bookTitle}>{book.title}</Text>    
+              <Text style={styles.bookAvailability}>Available</Text>    
```

Comment:
@guidoprinc this `Available` is ok? or should it be in a strings file or something?

Reply:
this can be in the json file you are using @patofernandez